Formula/t/teslamate.rb

```diff
@@ -0,0 +1,119 @@
+class Teslamate < Formula
+  desc "Self-hosted data logger for your Tesla"
+  homepage "https://docs.teslamate.org"
+  url "https://github.com/teslamate-org/teslamate/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "fe29dab0dd0b96bafe003b063a7f6f95338d64b26c7187a68bacd2023e423f82"
+  license "MIT"
+
+  depends_on "node" => :build
+  depends_on "postgresql@17" => :test
+  depends_on "elixir"
+  depends_on "erlang"
+
+  def install
+    # See https://docs.teslamate.org/docs/installation/debian/
+    system "mix", "local.hex", "--force"
+    system "mix", "local.rebar", "--force"
+    system "mix", "deps.get", "--only", "prod"
+    system "npm", "install", "--prefix", "./assets", *std_npm_args(prefix: false)
+    system "npm", "run", "deploy", "--prefix", "./assets"
+
+    with_env("MIX_ENV" => "prod") do
+      system "mix", "do", "phx.digest,", "release", "--overwrite"
+    end
+
+    touch buildpath/"teslamate.env"
+    etc.install "teslamate.env"
+    libexec.install Dir["_build/prod/rel/teslamate/*"]
+    bin.install_symlink Dir["#{libexec}/bin/teslamate"]
+
+    # Corresponds to https://github.com/teslamate-org/teslamate/blob/main/entrypoint.sh
```

Comment:
Then we should ship that instead of writing a custom one

Reply:
It doesn't feel right to use a Docker entrypoint script as a way to start a native application. It also assumes a specific working directory.

Reply:
I've stripped it down to just three lines (+shebang and options), I hope that's acceptable?

Reply:
@IngmarStein I wonder maybe we should upstream this script (I can see this would help with other package managers for building the recipe), can you do that? Thanks!